scripts/pre_commit_linter.py

```diff
@@ -401,7 +423,7 @@ def _get_data(cls, filepath, mode):
         if key not in cls._CACHE_DATA_DICT:
             with cls._get_cache_lock(key):
                 if key not in cls._CACHE_DATA_DICT:
-                    with open(filepath, mode) as f:
+                    with io.open(filepath, mode, encoding='utf-8') as f:
```

Comment:
I thought we were going to do this as: `python_utils.open`?

Reply:
I think we should really use io.open instead everywhere otherwise it will only get more complicated. For example the parameters that open and io.open use are different and a lot of files would need to be open as `encoding='utf-8'` that open does not support. So I really don't see the point of using Python 2's open anymore when io.open is clearly better and consistent through the two Python versions

Reply:
That's fine, since I don't think we do anything complicated with `open` in the codebase. As long as future contributors know they need to use `python_utils.open` instead of the regular `open`, everything should work out (right?)

Reply:
done